site/cfg.php

```diff
@@ -23,7 +23,7 @@
 $PWECore->setXMLDirectory($PWECore->getDataDirectory());
 $PWECore->setTempDirectory($tempdir);
 
-if (!is_dir("/home/gettauru")) {
+if (!is_dir('/app')) {
```

Comment:
change it into `!is_dir() && !is_dir()` to work on old+new

Reply:
fixed